app/views/transient_registrations/show.html.erb

```diff
@@ -70,6 +70,20 @@
         <% end %>
       </div>
     <% end %>
+
+    <div class="panel wcr-panel wcr-panel-border-all">
+      <h2 class="heading-medium">
+        <%= @transient_registration.company_name %>
+      </h2>
+      <p class="lede">
+        <%= t(".tier.#{@transient_registration.tier.downcase}") %> - <%= show_translation_or_filler(:registration_type) %>
+      </p>
+      <p>
+        <%= t(".labels.expires", formatted_date: display_expiry_date) %>
+        <br>
+        <%= t(".labels.account", email: @transient_registration.account_email) %>
```

Comment:
You might want to use the `show_translation_or_filler()` helper here. It shouldn't happen but seeing as we just ran a data fix today for it, it is possible that a non-AD registration does not have an account email 😩 

```suggestion
        <%= t(".labels.account", email: show_translation_or_filler(:account_email) %>
```

Reply:
No this won't work. that method has expectations on translations, don't just return a value or "-". I think I am going to leave it as it is, if there is no email it will just show `Account:`, no error is thrown, and then we can discuss with Martin if we should drop the line entirely :)

Reply:
Though you lost a bracket when you reverted the change!

Latest is showing as `<%= t(".labels.account", email: @transient_registration.account_email %>`

Reply:
yep! Just added it!! Sorry!